mod-permissions/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>folio-integration-tests</artifactId>
+        <groupId>org.folio</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>mod-permissions</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <cucumber.reporting.version>3.8.0</cucumber.reporting.version>
+        <testrail-api-java-client.version>2.0.1</testrail-api-java-client.version>
```

Comment:
Where/how are these properties used?

Reply:
I think both of these lines can be safely removed. The cucumber reports still get generated by karate whether this reference is there. Only about half the projects include the testrail java client. May be needed depending on what you want to do, but after removing both, tests still run and reports still get generated.